WrightTools/data/_data.py

```diff
@@ -531,7 +531,9 @@ def convert(self, destination_units, *, convert_variables=False, verbose=True):
                         )
         self._on_axes_updated()
 
-    def create_channel(self, name, values=None, shape=None, units=None, **kwargs) -> Channel:
+    def create_channel(
+        self, name, values=None, shape=None, units=None, *, dtype=None, **kwargs
```

Comment:
I added the `*` for dtype to be kwarg only.
I chose that location for strict backwards compatibility reasons.
That said, given that shape is ignored if values are given, I think it is nonsensical to have shape (or units, for that matter) be accessible only when an ignored (or default) value is given.

Therefore, I move that we place the `*` between values and shape, maintaining the ability to simply pass the values, but making the others keyword only.

Same applies to `create_variable` below